src/sagemaker/tensorflow/serving.py

```diff
@@ -128,10 +129,17 @@ def prepare_container_def(self, instance_type, accelerator_type=None):
         env = self._get_container_env()
 
         if self.entry_point:
-            model_data = sagemaker.utils.repack_model(self.entry_point,
-                                                      self.source_dir,
-                                                      self.model_data,
-                                                      self.sagemaker_session)
+            key_prefix = sagemaker.fw_utils.model_code_key_prefix(self.key_prefix, self.name, image)
```

Comment:
I see there are unit tests added in test_mxnet, but do we need to have unit test for serving.py and model.py class as well?